processor/k8sprocessor/options.go

```diff
@@ -135,15 +135,22 @@ func extractFieldRules(fieldType string, fields ...FieldExtractConfig) ([]kube.F
 	rules := []kube.FieldExtractionRule{}
 	for _, a := range fields {
 		name := a.TagName
+
+		switch a.From {
+		// By default if the From field is not set for labels and annotations we want to extract them from pod
+		case "", kube.MetadataFromPod:
+			a.From = kube.MetadataFromPod
+		case kube.MetadataFromNamespace:
+			a.From = kube.MetadataFromNamespace
+		default:
+			return rules, fmt.Errorf("from must contain exactly one value like pod or namespace")
```

Comment:
nit: "like pod or namespace" suggests it accepts other values that are like these such as deployments. It is also helpful to print out the user specified value so it is easy to spot the error in config in large and complex files. 

I think this should read something like: `"%s" is not a valid choice for `From`. Must be one of: "pod", "namespace"` where `%s` gets replaced with the user specified value.